tests/article.test.js

```diff
@@ -11,6 +11,12 @@ const { expect } = chai;
 before(userSeeder.addUserToDb);
 
 let userToken;
+// const publish2 = {
```

Comment:
```suggestion
// const publish2 = {
```
#### Nitpick
If it's not needed, remove it. Don't comment it out.

Reply:
I have removed it